Traceback (most recent call last):
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\Lib\tkinter\__init__.py", line 1968, in __call__
    return self.func(*args)
           ^^^^^^^^^^^^^^^^
  File "C:\Users\Antonio\Documents\GitHub\SDI_Tools\SpecTool.py", line 935, in <lambda>
    add = tk.Button(root, text= "Add Entry", command=lambda:AddEntry(con,treeview,messageBox))
                                                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Antonio\Documents\GitHub\SDI_Tools\SpecTool.py", line 831, in AddEntry
    db.addEntry([splitfile[0].strip(), splitfile[1].strip(), splitfile[2].split('.docx')[0].strip(), f], cur)
  File "C:\Users\Antonio\Documents\GitHub\SDI_Tools\SpecDB.py", line 58, in addEntry
    #Add Entry
sqlite3.OperationalError: near "s": syntax error
